crates/re_types/definitions/rerun/blueprint/archetypes/container_blueprint.fbs

```diff
@@ -32,7 +32,9 @@ table ContainerBlueprint (
     /// For `Horizontal`/`Vertical` containers, the length of this list should always match the number of contents.
     primary_weights: rerun.blueprint.components.PrimaryWeights ("attr.rerun.component_optional", nullable, order: 400);
 
-    /// The weights of the secondary axis. For `Grid` this is the row weights. Ignored for `Horizontal`/`Vertical` containers.
+    /// The weights of the secondary axis. For `Grid` this is the row weights.
+    ///
+    /// Ignored for `Horizontal`/`Vertical` containers.
     secondary_weights: rerun.blueprint.components.SecondaryWeights ("attr.rerun.component_optional", nullable, order: 500);
```

Comment:
Didn't we agree that `horizontal` and `vertical` was more intuitive than `primary/secondary` ?

Reply:
Ahh right -- forgot to make that change after we had discussed it.  Will do that in a followup PR.